plugin/fzf.vim

```diff
@@ -537,6 +549,15 @@ function! s:execute(dict, command, use_height, temps) abort
       call jobstart(cmd, fzf)
       return []
     endif
+  elseif has('win32unix')
+    if $TERM ==# 'cygwin'
```

Comment:
I'm going to merge this to devel branch first and then merge to master when I release 0.16.8, so let me know if there are other things that can be done on the binary side that can help simplyfing the Vim plugin.